src/core_plugins/getting_started/public/components/getting_started/getting_started.js

```diff
@@ -0,0 +1,63 @@
+import { uiModules } from 'ui/modules';
+import uiChrome from 'ui/chrome';
+import 'ui/storage';
+import 'ui/filters/trust_as_html';
+import { gettingStartedRegistry } from 'ui/getting_started_registry';
+import { GETTING_STARTED_OPT_OUT } from '../../lib/constants';
+
+import kibanaLogo from 'ui/images/logo-kibana-small.svg';
+import beatsLogo from 'ui/images/logo-beats-small.svg';
+import logstashLogo from 'ui/images/logo-logstash-small.svg';
+import dashboardIcon from 'ui/images/icon-dashboard.svg';
+import shieldIcon from 'ui/images/icon-shield.svg';
+
+import template from './getting_started.html';
+import './getting_started.less';
+
+const app = uiModules.get('kibana');
+
+app.directive('gettingStarted', function ($injector) {
+
+  const localStorageService = $injector.get('localStorage');
+
+  return {
+    restrict: 'E',
+    template: template,
+    scope: {
+    },
+    bindToController: true,
+    controllerAs: 'gettingStarted',
+    controller: class GettingStartedController {
+      constructor() {
+        if (this.hasOptedOut) {
+          uiChrome.setVisible(true);
+        } else {
+          uiChrome.setVisible(false);
+        }
+
+        this.topMessage = gettingStartedRegistry.topMessage;
+        this.manageAndMonitorMessages = gettingStartedRegistry.manageAndMonitorMessages.join(' ');
+        this.imageUrls = {
+          kibanaLogo,
+          beatsLogo,
+          logstashLogo,
+          dashboardIcon,
+          shieldIcon
+        };
+      }
+
+      get hasManageAndMonitorMessages() {
+        return this.manageAndMonitorMessages.length > 0;
+      }
+
+      get hasOptedOut() {
```

Comment:
Call me old-fashioned, but because of the way Angular's two-way binding works, I'm always a little leery about using native getters and setters on the scope. From an interface perspective, it's not clear that you're actually calling a method, so if someone ever bound to this "property", they're not only be incurring a more significant performance cost than expected, but they'd also get an error if they tried to two-way bind.

I think this would be a little simpler and more maintainable if we just made this a method:

```javascript
hasOptedOut() {
  /* ... */
}
```

I find this much easier to read from the call-site because I immediately understand we're going to do some logic to derive or extract this value, and we're not just reading from a pre-calculated property:

```
if (this.hasOptedOut()) {
  /* ... */
}
```

What do you think?